command/ibft/candidates/result.go

```diff
@@ -7,11 +7,13 @@ import (
 	"github.com/0xPolygon/polygon-edge/command/helper"
 	ibftHelper "github.com/0xPolygon/polygon-edge/command/ibft/helper"
 	ibftOp "github.com/0xPolygon/polygon-edge/consensus/ibft/proto"
+	"github.com/0xPolygon/polygon-edge/helper/hex"
 )
 
 type IBFTCandidate struct {
-	Address string          `json:"address"`
-	Vote    ibftHelper.Vote `json:"vote"`
+	Address      string          `json:"address"`
+	BLSPublicKey *string         `json:"bls_pubkey"`
```

Comment:
Why do you need this if you are not using this anywhere. We should either add BLSPublicKey in output or remove this variable

Reply:
Removed because it's not necessary, I think 17453468b3d60073eabb2830bc3301da560c4f19